local_planner/src/local_footstep_planner.cpp

```diff
@@ -84,8 +84,9 @@ void LocalFootstepPlanner::getFootPositionsBodyFrame(
 }
 
 void LocalFootstepPlanner::computeContactSchedule(
-    int current_plan_index, const Eigen::VectorXi &ref_primitive_plan,
-    int control_mode, std::vector<std::vector<bool>> &contact_schedule) {
+    int current_plan_index, const Eigen::MatrixXd &body_plan,
+    const Eigen::VectorXi &ref_primitive_plan, int control_mode,
+    std::vector<std::vector<bool>> &contact_schedule) {
```

Comment:
may I suggest to change this to 
`std::vector<std::array<bool,4>> &contact_schedule`

Reply:
Hey, glad you're curious about Quad-SDK - PlotJuggler has been a huge help for our online debugging!

There are certainly a lot of little refactors like this that we could do all over the repo so I'm not sure we'll get to this one. We've also defaulted to using vectors for most containers just to keep the types relatively consistent (and to make it easier in case folks eventually want to change the number of legs on the platform).

Reply:
I am VERY curious :wink: 
I will definitively keeping an eye on this project.

Cheers